setup.py

```diff
@@ -46,15 +45,16 @@
     ],
     'pipeline.module_versions': [
         'aodndata = aodndata.version:__version__',
-        'cc-plugin-imos = cc_plugin_imos:__version__'
+        'cc-plugin-imos = cc_plugin_imos:__version__',
+        'ncwriter = ncwriter:__version__'
     ]
 }
 
 INSTALL_REQUIRES = [
     'aodncore>=0.24.0',
     'cc-plugin-imos>=1.3.0',
     'matplotlib==1.5.1',
-    'ncwriter>=0.1.0',
+    'ncwriter>=0.2.0',
```

Comment:
as a reminder, we need to push the new version on jenkins

Reply:
Thanks, will do. It's already there at the systest stage. Once this PR is merged and we push it to systest also, then we can test both in RC before pushing to prod.